packages/react/src/components/Authenticator/index.tsx

```diff
@@ -23,6 +25,10 @@ export function Authenticator({
 
   const [state, send] = useActor(service);
 
+  React.useEffect(() => {
+    I18n.putVocabularies(translations);
+  }, []);
```

Comment:
I was worried that there would be an English-only render and a French render when the state changes.

Interestingly, this effect is called prior to rendering (like `componentWillMount`) so the Authenticator renders correctly the 1st time!

Reply:
👍

Reply:
TIL, I'm not sure I would have expected that. Thanks for calling this out!

Reply:
I was surprised as well! But the important take-away is that **we want to avoid side-effects from `import`ing**.

Meaning, `I18n.putVocabularies` **should not** be at the same level as `export...` or `import...`, because it creates side-effects that tree-shaking (particularly Angular) doesn't support well.